pulse_actions/handlers/config.py

```diff
@@ -10,6 +11,9 @@
     "exchange/treeherder/v1/resultset-actions": {
         "resultset_actions": treeherder_resultset.on_resultset_action_event
     },
+    "exchange/treeherder/v1/resultset-runnable-job-actions": {
```

Comment:
Could you please add a comment for each of the actions so we know what each one is for? It's rather hard to understand what a runnable action from TH is.

Thanks!